Kudu.Core/Helpers/PostDeploymentHelper.cs

```diff
@@ -287,11 +290,17 @@ private static async Task<bool> TryFunctionsRuntimeSyncTriggers(string requestId
 
                 return true;
             }
-            catch (Exception)
+            catch (Exception ex)
             {
-                Trace(TraceEventType.Information, "Syncing function triggers by calling the functions runtime site failed.");
+                exception = ex;
                 return false;
             }
+            finally
+            {
+                Trace(TraceEventType.Information,
+                      "Syncing function triggers by calling the functions runtime site failed.",
```

Comment:
I am bit confused. The second parameter has hardcoded `failed` message. The third parameter can be `failed` or `successful`.

Reply:
Thanks for catching it. Fixed it.